<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 0)</string>
  </property>
  <widget class="QLabel" name="BglabelQR">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-140</y>
     <width>1931</width>
     <height>1151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/o/Downloads/bg_edited.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>bg_edited.png</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="frameQR">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>581</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <widget class="QLabel" name="labelNo1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>32</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 32pt &quot;Times New Roman&quot;;


</string>
    </property>
    <property name="text">
     <string>You have successfully </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNo2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>311</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>30</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 30pt&quot;Times New Roman&quot;;
border:10px solid white

</string>
    </property>
    <property name="text">
     <string>Registered!</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQRstate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>371</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 14pt &quot;Times New Roman&quot;;
border:10px solid white
</string>
    </property>
    <property name="text">
     <string>Here is your QR code for Attendace :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateQR">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: 16pt &quot;Times New Roman&quot;;
background-color:rgb(0, 0, 0);
border-radius:10;


}

QPushButton:hover{
background-color:#4a569d;


}</string>
    </property>
    <property name="text">
     <string>Generate QR code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadQR_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>610</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 255, 255);
font: 16pt &quot;Times New Roman&quot;;
background-color:rgb(0, 0, 0);
border-radius:10;

}

QPushButton:hover{
background-color:#4a569d;

}</string>
    </property>
    <property name="text">
     <string>Download QR</string>
    </property>
   </widget>
   <widget class="QLabel" name="qr_qr">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>291</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(192, 72, 72)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="back6_butt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>&lt;   BACK</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img/t.qrc"/>
 </resources>
 <connections/>
</ui>
